configs/sites/tier2/aws-rocky8/README.md

```diff
@@ -230,6 +230,7 @@ spack external find --scope system \
     --exclude openssh --exclude python
 spack external find --scope system wget
 spack external find --scope system mysql
+spack external find --scope system grep
```

Comment:
Have you tested this? I had trouble getting `find external` to identify grep and I had to add it to the site config manually. That said, my problems might have been my own.

Reply:
Did I not fix the `find external grep` recently?

Reply:
maybe! my experience was from like 3 weeks ago and might have been from a branch slightly further out of date than that. Testing this would take more effort than its worth and I'll take it on faith that its working. 

Steve feel free to resolve this.

Reply:
I'm double checking this on the instances I created to make sure, but I saw the PR merged.